### ISSUE TITLE
[en] Goin', walkin' etc: ignore spelling and add a style rule

### ISSUE THREAD

**Issue by MikeUnwalla** [languagetool-org/languagetool#2320]
Noted during the correction for #2311.

The words that are tagged as VBG (goin', Puttin', smilin', laughin' etc) all give spelling errors.

Add a disambiguation rule to ignore the spelling. Add a grammar rule to say that the text is informal and suggest the full form of the present participle.

**Comment by danielnaber** [comment]
I guess this can be done with any verb? The correct solution would require to also add the words to `added.txt` (to avoid false alarm in unpaired brackets rule), but as there are so many, this would need to be done in code.

Here are the most common `...in'` forms on Google books, BTW:

```
866266  goin
744783  actin
357244  doin
356351  dentin
310521  comin
259937  lookin
252497  gettin
249810  fuckin
226054  bein
210106  muffin
145739  bobbin
135249  cyclin
126863  makin
123546  agin
104238  livin
99616   havin
94819   darin
85567   feelin
79415   matin
61883   playin
60644   fightin
55635   jerkin
55459   givin
54110   askin
51887   eatin
50730   hangin
47782   motherin
47671   meetin
```

**Comment by MikeUnwalla** [comment]
> I guess this can be done with any verb? 

Yes. (When I was disambiguatin', I found a few problems [grin].)

Also, some of the 'verbs' are nouns. Examples: muffin, bobbin, jerkin.

**Comment by MikeUnwalla** [comment]
I wonder. Why do we have disambiguation rule VBG_IN? It adds VBG to these abbreviated words. But, as best I know, other disambiguation rules and grammar rules do not use `~in/VBG+apostrophe` in their patterns.

Possibly, my suggestion to ignore spelling and add a grammar rule is not so sensible. Maybe a better option is to continue to give a spelling warning, and also have a rule (with higher priority) that suggests the full form.

**Comment by danielnaber** [comment]
> Also, some of the 'verbs' are nouns. Examples: muffin, bobbin, jerkin.

Yes, but only because LT know `muffing` etc. as VBG.
